internal/api/web/views/components/dashboard_components/dashboard_home_page/quiz_tile.templ

```diff
@@ -0,0 +1,25 @@
+package dashboard_components
+
+import "github.com/Molnes/Nyhetsjeger/internal/data/quizzes"
+
+templ QuizTile(quiz quizzes.Quiz) {
+	<a
+		class="mr-4 mb-4 bg-gray-200 shadow-md rounded-lg max-w-56 overflow-hidden flex flex-col hover:shadow-lg transition duration-100 ease-in-out transform hover:-translate-y-1 hover:scale-105"
+		href={ templ.SafeURL("/dashboard/quiz/edit/" + quiz.ID.String()) }
+	>
+		// src={ quiz.ImageURL.String() }
+		<img
+			src="https://picsum.photos/200/300"
+			alt={ quiz.Title }
+			class="h-48 w-full bg-gray-500 "
+		/>
+		<div class="py-2 pl-1 pr-4">
```

Comment:
Why is there more padding on the right side than the left? Seems like a strange choice.

Reply:
175078ee4d9dd21b0cf00c48a7d7c5443dabb8fb